docs/spec/enums.rst

```diff
@@ -0,0 +1,319 @@
+Enumerations
+============
+
+The ``Enum`` class behaves differently from other Python classes in several 
```

Comment:
Maybe too nitpicky, but maybe make it explicit that you are referring to stdlib's `enum.Enum`, and not just any old `Enum` class?

Reply:
Yeah, agreed. I should also make it clear that this applies to subclasses of `enum.Enum` and any class that uses `enum.EnumType` (or a subclass thereof) as a metaclass.